GraphAlgorithms/Graphs/GraphLoader.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace GraphAlgorithms.Graphs
+{
+    public class GraphLoader
+    {
+        private int currentLine;
+
+        private readonly string[] lines;
+
+        private GraphLoader(string[] lines)
+        {
+            this.lines = lines;
```

Comment:
_lines

Reply:
can you add .editorconfig? I tried to add one from VS but there's a bug preventing me from adding this on the solution level